specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4199,6 +4230,16 @@
         "recognition_01",
         "recognition_02"
       ]
+    },    
+    "returnRecognitionModel": {
+      "name": "returnRecognitionModel",
+      "description": "Whether to return the 'RecognitionModel' required for the current operation.",
```

Comment:
`"description": "A value indicating whether the operation should return 'recognitionModel' in response.",`

Reply:
Changed.